.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   node:
     docker:
-      - image: circleci/node:10.15
+      - image: cimg/node:10.21
```

Comment:
This change has been extracted to https://github.com/firefox-devtools/profiler-server/pull/85, it's in this PR just so that the tests pass.